lib/dune

```diff
@@ -4,10 +4,16 @@
  (enabled_if (= %{system} macosx))
  (action (copy %{deps} %{target})))
 
+(rule
+ (deps sandbox.jail.ml)
+ (target sandbox.ml)
+ (enabled_if (= %{system} freebsd))
+ (action (copy %{deps} %{target})))
+
 (rule
  (deps sandbox.runc.ml)
  (target sandbox.ml)
- (enabled_if (<> %{system} macosx))
+ (enabled_if (and (<> %{system} macosx) (<> %{system} freebsd)))
```

Comment:
Given `runc` is Linux can this rule become `(enabled_if (<> %{system} linux))`?

Reply:
That would be `(enabled_if (= %{system} linux))`, but yes, I suppose this could work.

Reply:
Well, it turns out making this change breaks a few builds (such as debian 11 on arm32 and powerpc), so I'll revert that particular change.

Reply:
The issue is to do with the variable expansion in `dune` if you add a default `sandbox.dummy.ml` with
```ocaml
type config = unit

let cmdliner = failwith "Sandbox not available"
```
and setup a copy rule for `Win32` that should work.  I'm not sure what is happening with the POWER build, the reported value for `system` is clearly not doing what I expect.